examples/with-cron/docker-compose.yml

```diff
@@ -105,5 +106,6 @@ services:
       DOLI_CRON_KEY: ${DOLI_CRON_KEY:-mycronsecurekey}
       DOLI_INIT_DEMO: ${DOLI_INIT_DEMO:-0}
       DOLI_COMPANY_NAME: ${DOLI_COMPANY_NAME:-MyBigCompany}
+      DOLI_INSTANCE_UNIQUE_ID: $DOLI_INSTANCE_ID # Add a random ASCII 16 char string here
```

Comment:
I would remove the comment so that the user perform the change in the .env directly